io.bit3.WhatsAppQT.yml

```diff
@@ -0,0 +1,34 @@
+app-id: io.bit3.WhatsAppQT
+runtime: org.kde.Platform
+runtime-version: '5.14'
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp
+base-version: '5.14'
+command: WhatsAppQT
+finish-args:
+- "--share=ipc"
+- "--socket=x11"
```

Comment:
Should be `fallback-x11`, assuming it actually works natively on Wayland.

Reply:
I will change it and try it out.